tests/HttpRequestServiceTest.php

```diff
@@ -179,6 +279,8 @@ public function testJSONResponse()
         $result = $this->httpRequestServiceClass->json();
 
         $this->assertEquals(json_decode($responseJson, true), $result);
+
+        $result = $this->httpRequestServiceClass->getResponse();
```

Comment:
```suggestion
        $this->assertEquals(json_decode($responseJson, true), $result);
```

Reply:
@DenTray I changed this part to cover the getResponse method